### ISSUE TITLE
Tests on Travis

### ISSUE THREAD

**Issue by ain** [helpers/handlebars-helper-include#4]
I think also here on this helper we have the clear test mechanism missing, incl. integration with Travis.

@jonschlinkert how did you implement the tests (you have fixtures in place)?

**Comment by jonschlinkert** [comment]
this helper needs to be updated all around. I'll label it

**Comment by ain** [comment]
:+1:

**Comment by jonschlinkert** [comment]
Refactored, see: https://github.com/helpers/handlebars-helper-include/commit/6437c8c103568d9501b5f7d228c281d3602ac11e.

For the time being, I've decided not to spend time on unit testing the version that uses the old assemble. It's too much of a pita. We just finished with a ton of changes for assemble v0.6.0, and we're going to try to roll that out this week - and this helper will be pretty much useless with the new assemble (b/c helpers are easier and cleaner than this)
